### ISSUE TITLE
Progress gets unaligned when 100% happens at the end of the available dots

### ISSUE THREAD

**Issue by Slamdunk** [squizlabs/PHP_CodeSniffer#3058]
### Expected vs Actual output diff

```diff
 $ vendor/bin/phpcs
 ............................................................  60 / 120 (50%)
-............................................................  120 / 120 (100%)
+............................................................                                                             120 / 120 (100%)
```

**Comment by jrfnl** [comment]
Tested and confirmed.

![image](https://user-images.githubusercontent.com/663378/92314143-26b93a80-efd4-11ea-8941-d5500d162174.png)

**Comment by jrfnl** [comment]
PR #3106 should fix this.

**Comment by gsherwood** [comment]
Thanks for reporting this. I had fixed it in the 4.0 branch but forgot to cherry-pick that back into master.

For reference, original 4.0 commit was: https://github.com/squizlabs/PHP_CodeSniffer/commit/9c901a4bba15fe436d6e6c6d382fa11076c0e4c8
